bitnami/opencart/README.md

```diff
@@ -48,74 +48,147 @@ The command removes all the Kubernetes components associated with the chart and
+| `livenessProbe`                      | Liveness probe configuration                                                                                          | `Check values.yaml file`                       |
+| `nodeAffinityPreset.type`            | Node affinity preset type. Ignored if `affinity` is set. Allowed values: `soft` or `hard`                             | `""`                                           |
+| `nodeAffinityPreset.key`             | Node label key to match Ignored if `affinity` is set.                                                                 | `""`                                           |
+| `nodeAffinityPreset.values`          | Node label values to match. Ignored if `affinity` is set.                                                             | `[]`                                           |
+| `opencartHost`                       | OpenCart host to create application URLs (when ingress, it will be ignored)                                           | `nil`                                          |
+| `opencartUsername`                   | User of the application                                                                                               | `user`                                         |
+| `opencartPassword`                   | Application password                                                                                                  | _random 10 character long alphanumeric string_ |
+| `opencartEmail`                      | Admin email                                                                                                           | `user@example.com`                             |
+| `opencartFirstName`                  | First Name                                                                                                            | `Bitnami`                                      |
+| `opencartLastName`                   | Last Name                                                                                                             | `Name`                                         |
+| `opencartCookieCheckIP`              | Whether to check the cookie's IP address or not                                                                       | `no`                                           |
+| `opencartCountry`                    | Default country of the store                                                                                          | `us`                                           |
+| `opencartLanguage`                   | Default language of the store (iso code)                                                                              | `en`                                           |
+| `opencartSkipInstall`                | Skip OpenCart installation wizard (`no` / `yes`)                                                                      | `no`                                           |
```

Comment:
There are a few parameters that aren't used.

Reply:
good catch, removing